index.html

```diff
@@ -10047,7 +10039,7 @@ <h5>Presentational Role Inheritance</h5>
 						<td class="role-scope">
 							<ul>
 								<li><rref>tree</rref></li>
-								<li><rref>group</rref> owned by <rref>treeitem</rref></li>
+								<li><rref>group</rref> with <a>accessibility parent</a> <rref>treeitem</rref></li>
```

Comment:
Not really about this PR but a group with accessibility parent tree should be allowed, too.

Reply:
Done https://github.com/w3c/aria/issues/2014